### DISCUSSION TITLE
Welcome to Suggestion Discussions!

### DISCUSSION THREAD

**Discussion by YuriSizov** [godotengine/godot-proposals#3983] [#Announcements]
In an effort to help organize and streamline the proposal process we are opening the Discussions section of this repository.

From the beginning the Godot Improvement Proposals repository has been a home to both fully fleshed out feature proposals and open-ended discussions without much technical detail to act upon. Given that we are currently sitting on over 2 thousand open proposals, this makes it very hard to navigate the sea of ideas. This Discussions section comes to offer a space to hold free-form discussions and let ideas blossom before they can be turned into ready-to-implement enhancement proposals.

At the same time, the issues section of this repository is going to be trimmed. We will go through all the open proposals and assign them new labels to signify their actual proposal status, regarding both the feature itself and the proposed implementation. If a reported issue lacks a solid use case, a limitation that it tries to solve, or it doesn't provide enough technical detail to yet consider it for implementation, we will close the issue and suggest to open a discussion instead.

As always, feel free to leave your comments and thoughts on either proposal issues or discussions, and vote for ideas you support. This is the main driving force to define the future of the engine, and we couldn't do it without you!

**Comment by allenwp** [comment]
Can this be [pinned](https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions#pinning-a-discussion)? (Or maybe this is already pinned and I didn’t notice it, since there wasn’t any iconography visible to me.)

**Reply by YuriSizov** [reply]
It is pinned, yes

![image](https://user-images.githubusercontent.com/11782833/185414545-1f8af610-89b1-4a3d-8a13-485d3336b7aa.png)

**Reply by allenwp** [reply]
Oh! I see! 🤦‍♂️ This is a different pinned behaviour that I’m used to with other systems, such as message forums. I entirely ignored that top banner, not understanding it was a clickable link to a pinned discussion. 😬 My bad!

**Reply by YuriSizov** [reply]
Hehe. It would probably be more obvious if we had several of them, because they form a row.

**Reply by Zireael07** [reply]
I agree that I completely overlooked that top banner too!
